snap-gpf-ui/src/main/resources/org/esa/snap/core/gpf/docs/gpf/org_esa_snap_core_gpf_common_SubsetOp.html

```diff
@@ -54,6 +54,13 @@ <h2>Parameters</h2>
   <td>The list of source bands.</td>
   <td></td>
 </tr>
+<tr>
```

Comment:
Those files do not need to be modified. They are generated before the release or in between.
See class: org.esa.snap.core.gpf.doclet.OperatorDoclet